src/pages/about-us.tsx

```diff
@@ -38,20 +38,23 @@ export const AboutUsPageQuery = graphql`
       }
     }
 
-    allTeamJson {
+    allContentfulTeamMember {
       nodes {
         id
         name
         image {
-          childImageSharp {
-            gatsbyImageData(
-              width: 400
-              aspectRatio: 1
-              layout: CONSTRAINED
-              placeholder: DOMINANT_COLOR
-              formats: [AUTO, WEBP, AVIF]
-            )
-          }
+          gatsbyImageData(
+            width: 400
+            aspectRatio: 1
+            layout: CONSTRAINED
+            placeholder: DOMINANT_COLOR
+            formats: [AUTO, WEBP, AVIF]
+            # people on the photos are not always in the center.
+            # this option from the contentful image api tries to resolve that.
+            cropFocus: FACE
+            resizingBehavior: FILL
+            jpegProgressive: false
+          )
```

Comment:
oh, that's actually much less code than expected 🤩 Thanks for the content model too!
![Screenshot 2022-06-09 at 09 08 03](https://user-images.githubusercontent.com/1701755/172786316-8bd95f94-c167-455a-87ae-f37878845c96.png)

Reply:
> I'm unsure. I think by retrieving it locally we would enable them to be served by gatsby cloud again. On the other side contentful assets are also served through their own CDN (https://www.contentful.com/developers/docs/references/content-delivery-api/).

I couldn't judge the pros and cons really well here tbh as I'm pretty new to the gatsby world so I went with the version that I got running pretty quickly. Retrieving the images locally should be a change we can make pretty easily next :)